backend/repository/transaction_outbox.py

```
@@ -63,6 +63,23 @@ class TransactionOutboxRepository:
     def __init__(self, db: Any) -> None:
         # Tests expect collection named 'outbox'
         self.collection = db.outbox
+        self._in_memory_mode = hasattr(self.collection, '__class__') and 'InMemoryCollection' in self.collection.__class__.__name__
```

Ruff (0.12.2)

Line too long (131 > 100)

(E501)